packages/editor/src/components/post-visibility/label.js

```diff
@@ -24,8 +24,11 @@ export default function PostVisibilityLabel() {
  * @return {string} Post visibility label.
  */
 export function usePostVisibilityLabel() {
-	const visibility = useSelect( ( select ) =>
-		select( editorStore ).getEditedPostVisibility()
+	const visibility = useSelect(
+		( select ) => select( editorStore ).getEditedPostVisibility(),
+		[]
```

Comment:
Just a question: why adding the dependencies empty array?

Reply:
This is just a best practice. `useSelect` is similar to `useEffect`, dependencies need to be provided and empty array means same thing. Only re-select values when component mounts or store updates.